src/sentry/backup/imports.py

```diff
@@ -10,13 +11,30 @@
 from sentry.backup.helpers import EXCLUDED_APPS
 
 
-def imports(src, printer=click.echo):
+class OldImportConfig(NamedTuple):
+    """While we are migrating to the new backup system, we need to take care not to break the old
+    and relatively untested workflows. This model allows us to stub in the old configs."""
+
+    # Do we allow users to update existing models, or force them to only insert new ones? The old
+    # behavior was to allow updates of already included models, but we want to move away from this.
+    # TODO(getsentry/team-ospo#170): This is a noop for now, but will be used as we migrate to
+    # `INSERT-only` importing logic.
+    use_update_instead_of_create: bool = False
+
+    # Old imports use "natural" foreign keys, which in practice only changes how foreign keys into
+    # `sentry.User` are represented.
+    use_natural_foreign_keys: bool = False
```

Comment:
Should this be true, since old imports use natural foreign keys?

Reply:
It is `False` by default, but overridden to true in the old `src/sentry/runner/commands/backup.py` file also in this PR. The idea is that the default values are the ones we want for testing and new API endpoints, and the override values are only used once: at the old entry points that we'll be deprecating.

Reply:
Didn't catch the override, makes sense